packages/cli/src/handlers/preview.ts

```diff
@@ -218,7 +218,11 @@ export const startPreviewHandler = async (
         // Update
         console.error(`Updating project preview ${projectName}`);
         await deploy({ ...options, projectUuid: previewProject.projectUuid });
-        console.error(`Project updated on ${await projectUrl(previewProject)}`);
+        const url = await projectUrl(previewProject);
+        console.error(`Project updated on ${url}`);
+        if (process.env.CI === 'true') {
+            console.log(`::set-output name=url::${url}`);
```

Comment:
does `log` work ? or does it need to be error

Reply:
It displayed in my terminal 🤔 so I assume yes.
![Screenshot 2022-10-06 at 15 22 11](https://user-images.githubusercontent.com/9117144/194338211-bd0bd01b-efd9-4669-8c27-f70cdc9a7b92.png)

Reply:
@rephus perhaps it shows console.log when CI=true 💡